index.html

```diff
@@ -0,0 +1,1729 @@
+                    <h4 class="h5">
+                      <a href="#"
+                        >Matt Riddle Getting Called Up to 'Friday Night
+                        SmackDown'</a
+                      >
+                    </h4>
+                  </div>
+                  <div class="col-lg-4 col-sm-12 col-md-4">
+                    <div class="us-img">
+                      <img
+                        src="resources/wwe-smackdown-jeff-hardy-daniel-bryan.jpg"
+                        alt=""
+                      />
+                    </div>
+                    <h4 class="h5">
+                      <a href="#"
+                        >'Friday Night SmackDown' Results (5/29): IC Tournament
+                        Continues</a
+                      >
+                    </h4>
+                  </div>
+                  <div class="col-lg-4 col-sm-12 col-md-4">
+                    <article class="d-flex pb-2 border-bottom">
+                      <div class="photo-1">
+                        <img src="resources/boston-marathon.jpg" alt="" />
+                      </div>
+                      <h4 class="h3">
+                        <a href="#"
+                          >Boston Marathon Is Going Virtual, So What Does It
+                          Mean for Other Marathons?</a
+                        >
+                      </h4>
+                    </article>
+
+                    <article class="d-flex pt-2 pb-2 border-bottom">
+                      <div class="pic">
+                        <img
+                          class="pr-3"
+                          src="resources/kansas-city-chiefs-fans.jpg"
+                          alt=""
+                        />
+                      </div>
+                      <h4 class="h3">
+                        <a href="#"
+                          >NFL Home-field Advantage Could Be Wiped Out By
+                          Playing Behind Closed Doors</a
+                        >
+                      </h4>
+                    </article>
+
+                    <article class="d-flex pt-2">
+                      <div class="pic">
+                        <img
+                          class="pr-3"
+                          src="resources/adidas-face-mask-press-image.jpg"
+                          alt=""
+                        />
+                      </div>
+                      <h4 class="h3">
+                        <a href="#"
+                          >Where to Buy Running Face Masks From Adidas, Nike,
+                          the NBA and More</a
+                        >
+                      </h4>
+                    </article>
+                  </div>
+                </div>
+              </div>
+            </div>
+
+            <div class="subscribe-book mt-3 mb-4">
+              <div class="row container-fluid">
+                <div class="col-md-7 col-lg-8 col-sm-12 col-xs-12">
+                  <div class="block-title1 red">Subscribe</div>
```

Comment:
- [ ] This section should have two columns for medium devices. Kindly check that.

Source | Clone
-- | -- 
![image](https://user-images.githubusercontent.com/57414789/83581373-3cfeb880-a504-11ea-9eff-83b2d336c4d7.png) | ![image](https://user-images.githubusercontent.com/57414789/83581359-32dcba00-a504-11ea-83dd-49b03cf367cc.png)

- [ ] Kindly check the layout for small devices.

Source | Clone
-- | --
![image](https://user-images.githubusercontent.com/57414789/83581462-7e8f6380-a504-11ea-91f5-3427223a38b9.png) | ![image](https://user-images.githubusercontent.com/57414789/83581446-70d9de00-a504-11ea-94b9-48211894ff8b.png)